input/fsh/examples/example-slotpsindiv1.fsh

```diff
@@ -0,0 +1,18 @@
+
+Instance: ExampleSlotPSIndiv1
+InstanceOf: FrSlotAgregateur
+Description: "Exemple 1 ressource slot PS indiv"
+Usage: #example
+
+* meta.security = https://mos.esante.gouv.fr/NOS/TRE_R314-TypeCreneau/FHIR/TRE-R314-TypeCreneau#PUBLIC
+* serviceType[+].coding = $v3-ActCode#AMB
+* serviceType[+].coding = $v3-ActCode#VR
+* serviceType[+].text = "Visite de contrôle"
+* specialty[+].coding = https://mos.esante.gouv.fr/NOS/TRE_R38-SpecialiteOrdinale/FHIR/TRE-R38-SpecialiteOrdinale#SM54
+* specialty[+].coding = https://mos.esante.gouv.fr/NOS/TRE_R38-SpecialiteOrdinale/FHIR/TRE-R38-SpecialiteOrdinale#SM05
+* specialty[+].text = "Médecine générale (polyvalente)"
```

Comment:
A quoi correspond ce text ? Il engendre une erreur pcq il y a qu'un text alors qu'il y a un binding required :

No code provided, and a code is required from the value set 'fr-practitioner-specialty' (http://interopsante.org/fhir/ValueSet/fr-practitioner-specialty\|1.0.1)

Il faudrait obligatoirement utiliser des codes issus de la R38, et rajouter optionnellement les display associés au code plutôt que le texte

Reply:
(idem ensemble des SLOT)